### PR TITLE
#55 - upgrade laravel medialibrary 6 -> 7.6

### PR DESCRIPTION
**Pull request by TomBurch**
image

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -4,52 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
```
Comment (Freddo3000) [inline composer.lock:4]: Couldn't composer.lock be added to .gitignore?

Comment (TomBurch) [reply composer.lock:4]: It was already added to .gitignore, this branch was probably made before that

Comment (Freddo3000) [reply composer.lock]: Gotcha

**Comment by TomBurch** [comment]
This is only a partial upgrade (current version is 9.6.*) to allow laravel framework to be upgraded

Converted files need to be regenerated

**Review by BorderKeeper — APPROVED** [review]
lgtm needs local testing and manual deletion of config.php file in boostrap cache.

**Pull request merged**